### DISCUSSION TITLE
libp2p Day @ ETHDenver 2024

### DISCUSSION THREAD

**Discussion by dhuseby** [swift-libp2p/swift-libp2p#25] [General]
Hello! 👋🏻​

I hope you are happy and well. My name is Dave Grantham and I'm the new libp2p Community Architect. I'm excited to announce libp2p Day @ ETHDenver 🚀 on February 29th from 13:30 to 18:00 with a social event to follow.​

I invite you to attend. We plan to have presentations and community oriented activities and snacks 🍌🥨🍿. There will also be a social event with food 🍖 and drinks 🥂 after we conclude. Please come and meet the rest of the libp2p community! 🏘️

🚀 libp2p Day is a gathering of developers, researchers, and engaged community members focused on pushing the boundaries of decentralized peer-to-peer networks. This event is more than just another meetup at ETHDenver. This is where we will dig into turning nuts and bolts solutions and cutting edge research into a usable and robust peer-to-peer networking library for our mobile and ad hoc internet of today.

Please register [here](https://lu.ma/libp2pday-denver) so we can save a spot for you. 😎​

​Location:
- The Slate, Denver
- ​Thursday, February 29th 1:30 PM - 6 PM local time

​How YOU Can Participate:
- ​[Submit a talk proposal](https://forms.gle/nwdXq5yM267pLCRQ9) 💬
- ​[Sponsor libp2p Day](https://forms.gle/vqjyFXKVD5uBU98N9) 💵

Join one or our chat channels
- ​Slack: [#libp2p-community](https://filecoinproject.slack.com/archives/C06HV0D00E5)
- Matrix: [libp2p-community](https://matrix.to/#/#libp2p-community:matrix.org)
- ​Discord: [#community](https://discordapp.com/channels/1204447718093750272/1204448127986434118)
- ​Telegram: [libp2p@ETHDenver 2024](https://t.me/+9yzJ5EYmAWlkNzY5)

​Cheers! 🍻

**Comment by dhuseby** [comment]
@btoms20 I'm still looking for speakers to present at libp2p Day @ ETHDenver 2024. Are you going? I'm trying to get as many libp2p implementations represented as possible. Would you consider giving a talk about swift-libp2p? If you accept, I'm asking that every speaker fill out a short form so that I can keep everybody organized and finalize the schedule quickly: https://forms.gle/qdcQY8gxbEWu8wMu6

**Reply by btoms20** [reply]
Hey @dhuseby, I'm not going to be able to make it to ETHDenver this year unfortunately. Thanks for reaching out though!
